zsh-autosuggestions.zsh

```diff
@@ -810,7 +810,8 @@ _zsh_autosuggest_async_request() {
 
 	# There's a weird bug here where ^C stops working unless we force a fork
```

Comment:
Thanks for the quick PR. Not a huge deal, but can you update this comment to read something like "There's a bug in older versions where ^C ..."?